server/dive_utils/models.py

```diff
@@ -43,6 +44,23 @@ class Track(BaseModel):
     confidencePairs: List[Tuple[str, float]] = Field(default_factory=lambda: [])
     attributes: Dict[str, Any] = Field(default_factory=lambda: {})
 
+    @validator('features')
+    def validateFeatures(cls, v: List[Feature], values: dict):
+        if len(v) > 0:
```

Comment:
do we really allow a track that has no features?  It will pass validation but won't load.

Reply:
Yes, because there are several places in code where the track object is initialized before features are added, such as in the deserializers.  see `load_csv_as_tracks_and_attributes`.

Reply:
:facepalm: - was thinking primarily client side forgot about the creation of empty tracks during loading.